cli/src/pcluster/utils.py

```diff
@@ -333,3 +333,33 @@ def get_attr(obj, attributes, default=None):
         if obj is None:
             return default
     return obj
+
+
+def get_placement_group_for_compute_resource(queue, managed_placement_groups, compute_resource):
```

Comment:
Can we try to split this function in to some methods in `cluster_config.py` and `cluster_stack.py`

Below is a rough idea. The final implementation can be something different

In `SlurmQueue` in `cluster_config.py`:
```
def is_fully_defined_placement_group(compute_resource_name) -> bool:
    ...

def get_fully_defined_placement_group(compute_resource_name) -> str:
    ...
```
Then, put other logic in `cluster_stack.py`

Reply:
will do!